tests/driver.sh

```diff
@@ -403,6 +403,15 @@ int main()
 }
 EOF
 
+# #error ...
+try_output 1 "" << EOF
+#error This is an error directive
+int main()
+{
+    return 0;
+}
+EOF
+
```

Comment:
```
Error This is an error directive at source location 10937, IR index 2236
tests/driver.sh: line 16: Aborted                 "$SHECC" -o "$tmp_exe" "$tmp_in"
#error This is an error directive
int main()
{
    return 0;
}
exit code => 1
output =>
```
@jserv Should we keep this in test? We seem to assume the build should be passed before testing.

Reply:
> Should we keep this in test? We seem to assume the build should be passed before testing.

The proposed test case contradicts the assumptions of the current test suite by introducing scenarios that unexpectedly bypass the compilation process. Let's discard it.